src/blackboxTest/bash/TestGetRequest.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+TESTPATH=$1
+EXIT_CODE=0
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+WHITE='\033[0m'
+
+
+
+testResults() {
```

Comment:
This is actually more like `printTestResults` or `printSummary`, or something like that, right? The name currently suggests that it's actually running something.